include/cripts/Headers.hpp

```diff
@@ -199,21 +163,17 @@ class Header
     self_type &operator+=(const cripts::string_view str);
 
     // These specialized assignment operators all use the above
-    template <size_t N>
-    self_type &
-    operator=(const char (&str)[N])
-    {
-      return operator=(cripts::string_view(str, str[N - 1] ? N : N - 1));
-    }
+    // template <size_t N>
+    // self_type &
+    // operator=(const char (&str)[N])
+    //{
+    //  return operator=(cripts::string_view(str, str[N - 1] ? N : N - 1));
+    //}
```

Comment:
Maybe just remove this commented out block?

Reply:
So, I left it here, because at some point it was needed and useful, but it breaks some dependencies in this new code. Let me add a code here, for clarity.